cypress/e2e/tests/administration/jira-connection/jira_field_validations.test.ts

```diff
@@ -0,0 +1,47 @@
+import { login } from "../../../../utils/utils";
```

Comment:
Let's keep the folder name same as model . 
"jira" 

and test name does not need "jira" as it is under that folder